stable/pomerium/templates/authenticate-deployment.yaml

```diff
@@ -0,0 +1,155 @@
+      priorityClassName: {{ .Values.priorityClassName }}
+{{- end }}
+      containers:
+      - name: {{ .Chart.Name }}
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        args:
+{{- range $key, $value := .Values.extraArgs }}
+{{- if $value }}
+          - --{{ $key }}={{ $value }}
+{{- else }}
+          - --{{ $key }}
+{{- end }}
+{{- end }}
+        env:
+        - name: SERVICES
+          value: authenticate
+        - name: COOKIE_SECRET
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "pomerium.fullname" . }}
+              key: cookie-secret
+        - name: SHARED_SECRET
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "pomerium.fullname" . }}
+              key: shared-secret
+        - name: REDIRECT_URL
+          value: {{ .Values.authenticate.redirectUrl }}
+        - name: PROXY_ROOT_DOMAIN
+          value: {{ .Values.authenticate.proxyRootDomains }}
+        - name: IDP_PROVIDER
+          value: {{ .Values.authenticate.idp.provider }}
+        - name: IDP_CLIENT_ID
+          valueFrom:
+            secretKeyRef:
+              name:  {{ template "pomerium.fullname" . }}
+              key: idp-client-id
+        - name: IDP_CLIENT_SECRET
+          valueFrom:
+            secretKeyRef:
+              name:  {{ template "pomerium.fullname" . }}
+              key: idp-client-secret
+{{- if .Values.authenticate.idp.url }}
+        - name: IDP_PROVIDER_URL
+          value: {{ .Values.authenticate.idp.url }}
+{{- end }}
+
+{{- if .Values.config.cert }}
+        - name: CERTIFICATE
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "pomerium.fullname" . }}
+              key: certificate
+{{- end }}
+{{- if .Values.config.key }}
+        - name: CERTIFICATE_KEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "pomerium.fullname" . }}
+              key: certificate-key
+{{- end }}
+{{- if .Values.config.ca }}
+        - name: CERTIFICATE_AUTHORITY
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "pomerium.fullname" . }}
+              key: certificate-authority
+{{- end }}
+{{- range $name, $value := .Values.extraEnv }}
+        - name: {{ $name }}
+          value: {{ quote $value }}
+{{- end }}
+        ports:
+          - containerPort: 443
+            name: https
+            protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /ping
+            port: https
+            scheme: HTTPS
+        readinessProbe:
+          httpGet:
+            path: /ping
+            port: https
+            scheme: HTTPS
+        
+        
+        volumeMounts:
+        - mountPath: /etc/pomerium/
+          name: {{ template "pomerium.fullname" . }}-policy
+     
+      volumes:
+      - name: {{ template "pomerium.fullname" . }}-policy
```

Comment:
@desimone this doesn't need to be prefixed by the fullname because it is local to the deployment

Reply:
Is the updated correct?

Reply:
yep, exactly, and there is the same thing in authorize-deployment by the way